client/my-sites/email/email-providers-stacked-comparison/index.tsx

```diff
@@ -41,61 +43,72 @@ type EmailProvidersStackedComparisonProps = {
 	productsList?: string[];
 	requestingSiteDomains?: boolean;
 	shoppingCartManager?: any;
-	selectedSite?: Site | null;
+	selectedSite?: SiteData | null;
 	selectedDomainName: string;
 	source: string;
 	titanMailMonthlyProduct?: any;
 	gSuiteAnnualProduct?: any;
 };
 
-const recordTracksEventAddToCartClick = (
-	comparisonContext: string,
-	validatedMailboxUuids: string[],
-	mailboxesAreValid: boolean,
-	provider: string,
-	source: string,
-	userCanAddEmail: boolean,
-	userCannotAddEmailReason: any
-) => {
-	recordTracksEvent( 'calypso_email_providers_add_click', {
-		context: comparisonContext,
-		mailbox_count: validatedMailboxUuids.length,
-		mailboxes_valid: mailboxesAreValid ? 1 : 0,
-		provider: provider,
-		source,
-		user_can_add_email: userCanAddEmail,
-		user_cannot_add_email_code: userCannotAddEmailReason ? userCannotAddEmailReason.code : '',
-	} );
-};
-
 const EmailProvidersStackedComparison: FunctionComponent< EmailProvidersStackedComparisonProps > = (
 	props
 ) => {
 	const translate = useTranslate();
+	const [ billingPeriod, setBillingPeriod ] = useState( IntervalLength.MONTHLY );
```

Comment:
We should rename those variables to be more consistent, e.g.:

```suggestion
	const [ billingIntervalLength, setBillingIntervalLength ] = useState( IntervalLength.MONTHLY );
```